test/uk/gov/hmrc/tai/repositories/EmploymentRepositorySpec.scala

```diff
@@ -125,7 +127,7 @@ class EmploymentRepositorySpec extends BaseSpec {
 
           val sut = testRepository(
             rtiConnector = mockRtiConnector,
-            cacheConnector = mockCacheConnector,
+            taiCacheRepository = mockCacheConnector,
```

Comment:
This seems confusing to me that Repository = Connector?

Reply:
good spot, that should be changed. U the man

Reply:
cheers bruva 😎